srcpkgs/linux-firmware/template

```diff
@@ -73,5 +72,20 @@ linux-firmware-network_package() {
 		vmove usr/lib/firmware/ueagle-atm
 		vmove usr/lib/firmware/ti-connectivity
 		vmove usr/lib/firmware/dpaa2
+
+		# Move only non-Raspberry Pi related brcm firmware files.
+		# Cannont use vmove since one of the files has a space in it,
+		# so manually create destination folder and move files.
+		mkdir -p ${PKGDESTDIR}/usr/lib/firmware/brcm
+		find ${DESTDIR}/usr/lib/firmware/brcm/* ! -iname 'brcmfmac434??-sdio.*' \
+		    | while read f; do mv "$f" ${PKGDESTDIR}/usr/lib/firmware//brcm; done
```

Comment:
(superfluous `/`)

Reply:
I've updated the PR to do what @sgn suggested, so this is no longer relevant.